### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by ghost** [spacecowboy/Feeder#318] [Ideas]
Hello, I propose to add the ability to auto-translate RSS feeds into the selected language. This makes it much easier to understand information from various sources.

Some APIs for translation:
- [LingvaTranslate](https://github.com/TheDavidDelta/lingva-translate)
- [Mozhi](https://codeberg.org/Aryak/mozhi)
- [LibreTranslate](https://github.com/LibreTranslate/LibreTranslate)

**Comment by Jefficus** [comment]
Personally, I want to know if an article is being shown in its original language or an AI translation, so I'd rather see a "translate on demand" feature. I read a number of languages and like reading each headline in its native form, but when my language skills fail me, I'd like to be able to get that article converted to English without having to take a bunch of steps. Maybe a long press or swipe to activate? 

And rather than call an external API directly, doesn't Android support a translation intent that can refer the job to whichever translation app is installed? That would probably be a lot less work than babysitting the API call internally.

**Reply by spacecowboy** [reply]
Not to mention that external APIs cost money and this a free app
